design/one-pager-generic-datasource.md

```diff
@@ -0,0 +1,171 @@
+# Patch from Generic Datasources
+
+* Owner: Maximilian Blatt (maximilian.blatt@accenture.com, @mistermx)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## 1 Background
```

Comment:
```suggestion
## Background
```

Since the design docs and one pagers usually aren't very long, we don't really use index numbers.

Reply:
Removed the indices.